### PR TITLE
Feature/107059 edit decision

### PR DESCRIPTION
**Pull request by elielijah321**
**What is the change?**
Edit decision 

**Why do we need the change?**
Requirements

**What is the impact?**
Minimal

**Azure DevOps Ticket**
107059

### CODE REVIEW

File: ConcernsCaseWork/ConcernsCaseWork.Service/Decision/DecisionDto.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+﻿using ConcernsCaseWork.API.Contracts.Enums;
```
Comment (chrisdexnimble) [inline ConcernsCaseWork/ConcernsCaseWork.Service/Decision/DecisionDto.cs:1]: I think you need to merge with the decisions feature as this dto seems similar to one we have created in the api contracts folder. Have a word with Mike and he'll show you. It should work (I think)

File: ConcernsCaseWork/Service.TRAMS/Decision/DecisionDto.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json;
```
Comment (chrisdexnimble) [inline ConcernsCaseWork/Service.TRAMS/Decision/DecisionDto.cs:1]: I'm not sure what this is, shouldn't there only be one DTO, and used by the concerns api ? 
(in the contracts proj btw)

**Review by chrisdexnimble — CHANGES_REQUESTED** [review]

File: ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Decision/Add.cshtml.cs [hunk 3]
```diff
@@ -27,90 +29,78 @@ public class AddPageModel : AbstractPageModel
...
-
-		[BindProperty]
-		public bool DecisionTypeWriteOffRecoverableFunding { get; set; }
-
-		[BindProperty]
-		public bool DecisionTypeOtherFinancialSupport { get; set; }
-
-		[BindProperty]
-		public bool DecisionTypeEstimatesFundingOrPupilNumberAdjustment { get; set; }
+		public int NotesMaxLength => 2000;
```
Comment (chrisdexnimble) [inline ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Decision/Add.cshtml.cs:34]: Just to make things a bit easier. Do we need the max note length here ?
If we do, can we make the constants on the DTO public and reference them? That we can avoid changing this page if the length gets increased

**Review by chrisdexnimble — CHANGES_REQUESTED** [review]
1 small change please

File: ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Decision/Add.cshtml.cs (hunk 3 above)
Comment (mikestock-nimble) [reply ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Decision/Add.cshtml.cs:34]: Will check and will add if required

Comment (mikestock-nimble) [reply ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Decision/Add.cshtml.cs]: added constant

**Review by chrisdexnimble — APPROVED** [review]

**Pull request merged**